basics/a29.go

```diff
@@ -0,0 +1,21 @@
+// alias declarations
+
+package main
+
+import "fmt"
+
+func main() {
+	type A = string // defining a new data type A w
+	var a A
+	a = "foo"
+	a += "bar"
+	fmt.Println(a)
+	fmt.Printf("Type of a is %T", a) // this prints string
```

Comment:
This does print string but then why does it print 'foobaz' along with it?

I see: "Type of a is stringfoobaz"

Reply:
Missed to add `"fmt.Printf("Type of a is %T\n", a)"`